python/ray/air/config.py

```diff
@@ -128,9 +121,16 @@ class ScalingConfig:
     use_gpu: Union[bool, SampleRange] = False
     resources_per_worker: Optional[Union[Dict, SampleRange]] = None
     placement_strategy: Union[str, SampleRange] = "PACK"
+    # DEPRECATED
     _max_cpu_fraction_per_node: Optional[Union[float, SampleRange]] = None
 
     def __post_init__(self):
+        if self._max_cpu_fraction_per_node:
+            warnings.warn(
+                "_max_cpu_fraction_per_node is deprecated and will be removed in Ray 2.8.",  # noqa: E501
+                DeprecationWarning,
+            )
```

Comment:
@ericl I wasn't sure if there was a better way to do this, LMK if you think I should raise the warning or completely remove it from the API (and let it fail without a message) instead .

Reply:
Let's just remove it, since it was underscore/experimental.